psychopy/iohub/client/keyboard.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, ioe_array):
 
     @property
     def key(self):
-        return self._key
+        return str(self._key, 'utf-8')
```

Comment:
This will raise an error on Py2.7 (because can't convert to a bytes string with unicode encoding)
You could do:
```
from past.builtins import unicode
```
then
```
return unicode(self._key, 'utf-8')
```